src/Analysis/Ast/Impl/Analyzer/ProgressReporter.cs

```diff
@@ -20,8 +20,8 @@
 
 namespace Microsoft.Python.Analysis.Analyzer {
     internal sealed class ProgressReporter : IProgressReporter, IDisposable {
-        private const int _initialDelay = 50;
-        private const int _reportingInterval = 100;
+        private readonly static TimeSpan s_initialDelay = TimeSpan.FromMilliseconds(50);
```

Comment:
We don't use Hungarian prefixes

Reply:
so, just _initialDelay?

Reply:
In our convention, it'd be `InitialDelay` as it is a static value.

Reply:
FWIW I think this change is from #1596, so should be commented there.